_data/publications.yml

```diff
@@ -99,8 +119,40 @@
 
+- title: 'Serological surveys to estimate cumulative incidence of SARS-CoV-2 infection in adults (Sero-MAss study), Massachusetts, July–August 2020: a mail-based cross-sectional study'
+  slug: serological-surveys
+  authors: Teah Snyder, Johanna Ravenhurst, Estee Y Cramer, Nicholas G Reich, Laura Balzer, Dominique Alfandari, A Lover
+  journal: BMJ Open 
+  pdf: https://bmjopen.bmj.com/content/bmjopen/11/8/e051157.full.pdf
+  year: 2021
+  doi: https://doi.org/10.1136/bmjopen-2021-051157
+  keywords: forecasting, covid-19
+  abstract: >
+    Objectives 
+      To estimate the seroprevalence of anti-SARSCoV-2 IgG and IgM among Massachusetts residents and to better understand asymptomatic SARS-CoV-2 transmission during the summer of 2020.
+    Design 
+      Mail-based cross-sectional survey.
+    Setting 
+      Massachusetts, USA.
+    Participants 
+      Primary sampling group: sample of undergraduate students at the University of Massachusetts, Amherst (n=548) and a member of their household (n=231).
+      Secondary sampling group: sample of graduate students, faculty, librarians and staff (n=214) and one member of their household (n=78). All participants were residents of Massachusetts without prior COVID-19 diagnosis.
+      Primary and secondary outcome measures 
+        Prevalence of SARS-CoV-2 seropositivity. Association of seroprevalence with variables including age, gender, race, geographic region, occupation and symptoms.
+    Results 
+      Approximately 27 000 persons were invited via email to assess eligibility. 1001 households were mailed dried blood spot sample kits, 762 returned blood samples for analysis. 
+      In the primary sample group, 36 individuals (4.6%) had IgG antibodies detected for an estimated weighted prevalence in this population of 5.3% (95% CI: 3.5 to 8.0). 
+      In the secondary sampling group, 10 participants (3.4%) had IgG antibodies detected for an estimated adjusted prevalence of 4.0% (95% CI: 2.2 to 7.4). No samples were IgM positive. 
+      No association was found in either group between seropositivity and self-reported work duties or customer-facing hours. 
+      In the primary sampling group, self-reported febrile illness since February 2020, male sex and minority race (Black or American Indian/Alaskan Native) were associated with seropositivity. 
+      No factors except geographic regions within the state were associated with evidence of prior SARSCoV-2 infection in the secondary sampling group.
+    Conclusions 
+      This study fills a critical gap in estimating the levels of subclinical and asymptomatic infection. Estimates can be used to calibrate models estimating levels of population immunity over time, and these data are critical for informing public health interventions and policy.
+  
```

Comment:
So, here I'd make the abstract shorter.  Perhaps only include the Objectives and Conclusions? tagging @nickreich for his opinion on this.